.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# All markdown files
+*.md @xmarcalx @kalyazin @pb8o @sudanl0
+
+# But not the ones in docs/
+docs/*.md
+
+# Except these specific ones
+docs/getting-started.md @xmarcalx @kalyazin @pb8o @sudanl0
+docs/prod-host-setup.md @xmarcalx @kalyazin @pb8o @sudanl0
+
+# Also cover all "*-policy.md" documents
+**/*policy.md @xmarcalx @kalyazin @pb8o @sudanl0
```

Comment:
nit: I do not know if a * before .md would be safer as per previous comment (which unfortunately made it italic!) but I am ok also with this.

Reply:
Ah, sorry, done